lib/parallel.rb

```diff
@@ -215,6 +215,20 @@ def each(array, options={}, &block)
       array
     end
 
+    def any?(array, options={}, &block)
+      block ||= -> (v) { v }
+      !map(array, options.merge(:preserve_results => false)) do |*args|
```

Comment:
`#each` returns the original array, so I couldn't do that, I need the result of the `map` call